### PR TITLE
Cleanup and simplify the "Fulfilling Requests" page.

### PR DESCRIPTION
**Pull request by dwightjl**
View staged changes here: https://deploy-preview-208--dreamy-villani-0e9e5c.netlify.app/docs/fulfilling-requests

Changes for fulfilling-requests.md:

- Update the Gist link for ATestnetConsumer.sol. The previous Gist uses outdated import paths.
- Remove the callout for setting the Remix environment to Solidity. The language environment is set automatically when you open the Gist. **Environment** is a setting under **Deploy and Run**; not the Gists menu.
- Add alt text to images for accessibility.
- Clean up the requirements section to provide a full list of requirements rather that one requirement with many sub-requirements and one callout box.
- Convert some of the warnings into steps. Many of these warnings flow better as explicit steps in the process.

### CODE REVIEW

File: docs/Node Operators/fulfilling-requests.md [hunk 1]
```diff
@@ -4,384 +4,288 @@ date: Last Modified
...
-}
-[/block]
 
-[block:api-header]
-{
-  "title": "Deploy your own Oracle contract"
-}
-[/block]
-- Go to <a href="https://remix.ethereum.org/#gist=03a079b9055f42d993d0066d6f454c6f&optimize=true&version=soljson-v0.4.24+commit.e67f0147.js" target="_blank" rel="noreferrer, noopener">Remix</a> and expand the gist menu
+##Requirements
```
Comment (alexroan) [inline docs/Node Operators/fulfilling-requests.md:13]: This renders fine, but could we add a space in between the hashes and the text for conformity with the rest of the page, please?

```suggestion
## Requirements
```

File: docs/Node Operators/fulfilling-requests.md [hunk 2]
```diff
@@ -4,384 +4,288 @@ date: Last Modified
...
-    },
-    {
-      "code": "0x514910771AF9Ca656af840dff83E8264EcF986CA",
-      "language": "text",
-      "name": "Mainnet"
-    }
-  ]
-}
-[/block]
+##Deploy your own Oracle contract
```
Comment (alexroan) [inline docs/Node Operators/fulfilling-requests.md:24]:
```suggestion
## Deploy your own Oracle contract
```

File: docs/Node Operators/fulfilling-requests.md [hunk 3]
```diff
@@ -4,384 +4,288 @@ date: Last Modified
...
+
+1. Click Create Job and you'll be notified of the new JobID creation. Take note of this JobID as you'll need it later.
+
+    ![A screenshot showing that the job is created successfully.](/files/a7a2f0c-Screenshot_from_2018-11-02_08-42-22.png)
+
+1. Repeat this process for each of the jobs above.
+
+The address for each job auto-populates with the zero-address. This means the Chainlink node will listen to your Job ID from any address. You can also specify an address in the params object of the RunLog initiator for your oracle contract.
+
+##Create a request to your node
```
Comment (alexroan) [inline docs/Node Operators/fulfilling-requests.md:252]:
```suggestion
## Create a request to your node
```

**Review by alexroan — CHANGES_REQUESTED** [review]

**Comment by dwightjl** [comment]
Re-synced and ready to merge. All suggested changes are integrated.

**Comment by dwightjl** [comment]
Updated and merged. See the PR description for the current list of changes.

**Review by PatrickAlphaC — CHANGES_REQUESTED** [review]
Great! Just one comment.

File: docs/Node Operators/fulfilling-requests.md [hunk 4]
```diff
@@ -4,100 +4,96 @@ date: Last Modified
...
-- Going through the [Beginner Walkthrough](../beginners-tutorial) will help you obtain Testnet LINK and set up Metamask- [Run an Ethereum Client](../run-an-ethereum-client/) - [Running a Chainlink Node](../running-a-chainlink-node/)
+Before you begin this guide, complete the following tasks to make sure you have all of the tools that you need:
 
-> 👍 Make sure to fund your node address!
-> 
-> In the `keys` tab, you'll see at the bottom `Account Addresses`. The address of your node is the `regular` one. 
-> 
-> IN OLDER VERSIONS: Go to `configuration` in your node. You'll see `ACCOUNT_ADDRESS`. This is the address of your node. Send this address ETH. You can find testnet ETH on various [faucets](../link-token-contracts/). If you don't see `ACCOUNT_ADDRESS` there, check the 'Keys' tab and scroll down
+- Complete the [Beginner Walkthrough](../beginners-tutorial) to obtain Testnet LINK and set up MetaMask.
+- [Run an Ethereum Client](../run-an-ethereum-client/) that your Chainlink Node can access.
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply docs/Node Operators/fulfilling-requests.md]: Good feedback. I clarified the difference and linked to the external services section of the ../run-an-ethereum-client/ doc.

**Comment by dwightjl** [comment]
@PatrickAlphaC  @alexroan  This is ready to merge. Can you review it once more?

**Review by PatrickAlphaC — APPROVED** [review]

**Pull request merged**